server/remote/remote.go

```diff
@@ -79,6 +79,10 @@ type Remote interface {
 	// Hook parses the post-commit hook from the Request body and returns the
 	// required data in a standard format.
 	Hook(ctx context.Context, r *http.Request) (*model.Repo, *model.Build, error)
+
+	// OrgMembership returns if user is member of organization and if user
+	// is admin/owner in that organization.
+	OrgMembership(ctx context.Context, u *model.User, owner string) (member, admin bool, err error)
```

Comment:
What do you think about using your `membership` struct instead of the two booleans.  `owner string` should probably rather be `name string`.

```go
type membership struct {
	Member bool
	Admin  bool
}
```

Reply:
Refactored to have single method in membership service